src/org/project/ShoppingCLI.java

```diff
@@ -0,0 +1,91 @@
+package org.project;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+public class ShoppingCLI {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        RAM ram1 = new RAM("HyperX", 8, RamUnit.GB);
+        Processor processor1 = new Processor("Intel", "i7", 12, CoresUnit.GHz);
+        Accessory accessory1 = new Accessory("Etui", new BigDecimal(29), "Iphone 14 Pro", "Skórzane");
+        Product computer = new Computer(UUID.randomUUID(), "MacBook Air", new BigDecimal(1200), 10, processor1, ram1);
+        Product smartphone = new Smartphone(UUID.randomUUID(), "Iphone 14 Pro", new BigDecimal(4999), 50, Color.PINK, 2500, accessory1);
+        Product electronics = new Electronics(UUID.randomUUID(), "Samsung TV", new BigDecimal(5999), 20);
+
+        Product[] availableProducts = {computer, smartphone, electronics};
+
+        Cart cart = new Cart();
+
+        while (true) {
```

Comment:
kazdy krok interackji z uzytkownikiem powinien byc nowa metoda ktora to bedzie tutaj w tej klasie zaimplementowana a jej sygantura bedzie w interface